product-requirements-document.md

```diff
@@ -0,0 +1,255 @@
+* Codebases index page (like [publiccode.net/codebases/](/codebases))
+* Individual codebase overview (like [publiccode.net/codebases/signalen](/codebases/signalen.html))
+* Team index page (like [publiccode.net/who-we-are](/who-we-are))
+* Members and perhaps even a page about each member? [future]
+
+## Technical and functional requirements
+
+### Technology and architecture
+
+* Open and public: Aim for 100% open-source and public domain languages and tools. Temporarily fall back on closed tools only when there is consensus on the benefits.
+* Simpler and fewer: Use common and established tech whenever possible. For example, prefer CSS over SCSS, and avoid Javascript unless necessary.
+* Easy expansion: Beautiful design should not interfere with or overly complexify the process of adding and editing content. Templatization and continued automatic ingestion of markdown is essential.
+* Unified content, flexible display: Information display should be flexible, with few sources serving multiple endpoints. Continue to use Jekyll, Liquid and CSS media queries to enable one content set to serve multiple endpoints, including print/PDF.
+* Rich content: Add abilities for embedded video, social and GitHub hooks
+
+### Experience
+
+* Presentation modes: Information should be accessible via desktop and mobile web browser, social media, email, and pdf (digital or print)
+* Human accessible: [WCAG 2.1 Level A](https://www.w3.org/WAI/WCAG21/quickref/) (minimum). Accommodate the vision-impaired with screen-reader friendly, high contrast design. Avoid absolute text size units. Ensure proper keyboard-based navigability.
+* Linguistically accessible: Speak to an international audience by using standard, non-colloquial English. Speak to a non-technical audience by avoiding jargon, or defining it when necessary and helpful.
+* Technically accessible: Tools, languages and implementation style should prioritize procedures and code that is relatively easy to grasp and collaborate on.
+
+### Practices
+
+* Visible process: Bring tools and communication closer. Select for technologies which broadly facilitate viewing, insight, understanding, sharing and participation.
+* [Dogfood](https://en.wikipedia.org/wiki/Eating_your_own_dog_food): Contribute to open source tools, practices and culture by avoiding closed/commercial platforms (even if they are arguably more powerful).
+* Invite and reward feedback: Create and bond with collaborators by responding to their ideas, incorporating the good ones, and visibly crediting them.
+* Document well: As the new site is built out of the old, align on agreed-upon best practices, and document these patterns as part of the process. Enable future collaborators (and forgetful selves) to understand why decisions were made.
+
+## Design Approach
+
+### Basic process
+
+* Gather initial ideas and goals from team via workshops
+* Reference existing Github repos
```

Comment:
```suggestion
* Reference existing GitHub repos
```

Reply:
Thanks for catching so many typos, @Ainali!